packages/office-addin-dev-certs/src/install.ts

```diff
@@ -12,7 +12,7 @@ function getInstallCommand(caCertificatePath: string, machine: boolean = false):
    switch (process.platform) {
       case "win32":
          const script = path.resolve(__dirname, "..\\scripts\\install.ps1");
-         return `powershell -ExecutionPolicy Bypass ${script} ${machine ? "LocalMachine" : "CurrentUser"} '${caCertificatePath}'`;
+         return `powershell -ExecutionPolicy Bypass "& '${script}' ${machine ? "LocalMachine" : "CurrentUser"} '${caCertificatePath}'`;
```

Comment:
What does the `"&` here do? I'm unclear if the double-quote is escaping the ampersand and what this means for the Powershell command line?

Reply:
The single quotes around `{script}` should allow it to work even if the script path contains a space character, which I presume is the key part of the fix here.

Reply:
Single quotes didn’t work. Tried many variations and this is the only one I found that worked. Found it in stack overflow

Reply:
single quotes around {script} results in the following error:
Error: Unable to install the CA certificate. At line:1 char:91
+ ... dules\office-addin-dev-certs\scripts\install.ps1' CurrentUser 'C:\Use ...
+                                                       ~~~~~~~~~~~
Unexpected token 'CurrentUser' in expression or statement.

Regarding what the & means: https://stackoverflow.com/questions/22074507/what-does-the-symbol-in-powershell-mean

And it was this stack overflow article where I got the idea to use the &:  https://stackoverflow.com/questions/18537098/spaces-cause-split-in-path-with-powershell

Reply:
There is a mix of single and double-quotes. Should there also be the double-quote in front of the ampersand?

If this is normalized to use double-quotes and the ampersand on its own, I would expect:
```
return `powershell -ExecutionPolicy Bypass & "${script}" ${machine ? "LocalMachine" : "CurrentUser"} "${caCertificatePath}"`;
```

Could you let me know if that would be OK or not?

Reply:
yes, this works as well:
powershell -ExecutionPolicy Bypass & "${script}" ${machine ? "LocalMachine" : "CurrentUser"} '${caCertificatePath}'

I can make that change

Reply:
OK, and the last set of quotes should be double-quotes not single-quotes. (You probably just mistyped it here.)

Reply:
OK updating and pushing change